common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala

```diff
@@ -418,11 +422,24 @@ object PaParser extends ImageProcessor {
     val isPa = List(image.metadata.credit, image.metadata.source).flatten.exists { creditOrSource =>
       paCredits.contains(creditOrSource.toLowerCase)
     }
-    if (isPa) {
-      val restrictions = if (image.metadata.description.exists(_.contains(restrictionMessage))) {
-        Some(restrictionMessage)
-      } else None
-      image.copy(usageRights = Agency("PA", restrictions = restrictions))
+    if (isPa && image.metadata.description.exists(_.contains(restrictionMessage))) {
+        val firstLease = LeasesByMedia.build(List(MediaLease(
+          id = Some(UUID.randomUUID().toString),
+          startDate = Some(DateTime.now()),
+          endDate = Some(DateTime.now().plusMonths(1)),
+          access = AllowUseLease,
+          mediaId = image.id,
+          createdAt = DateTime.now(),
+          leasedBy = None,
```

Comment:
Would this ever go in (and work), maybe that field could have some generic “user” helping identify all on-ingest Lease applications? Something like `ingestion.robot` or something. I’m still checking with Editorial. But if it works, it could be hugely valuable to eg. add a 24hrs Deny leases to a future Alamy Live News processor (them pics should be free after 24hrs).

Thanks for checking that!